gluahttp.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/yuin/gopher-lua"
```

Comment:
Why did you remove this? That doesn't make any sense.

Reply:
This is done by autoformating for Golang in VSCode, sorry.